packages/discord.js/src/structures/ContextMenuCommandInteraction.js

```diff
@@ -31,6 +31,7 @@ class ContextMenuCommandInteraction extends CommandInteraction {
      * The type of the target of the interaction; either {@link ApplicationCommandType.User}
      * or {@link ApplicationCommandType.Message}
      * @type {ApplicationCommandType.User|ApplicationCommandType.Message}
+     * @deprecated Use {@link CommandInteraction.commandType} instead
```

Comment:
Imo you can remove it without deprecating as samver:major pr's are getting merged now.